### PR TITLE
[CD] Update release dependencies

### PR DESCRIPTION
**Pull request by rawlingsj**

### CODE REVIEW

**Comment by rawlingsj** [comment]
restest this please

**Comment by rawlingsj** [comment]
This is the stack trace after upgrading fabric8-ipaas to the latest release of fabric8..

```
16:16:52 test(io.fabric8.mq.MainTest)  Time elapsed: 0.343 sec  <<< ERROR!
16:16:52 java.lang.RuntimeException: Empty String or null supplied as string-default value for property VIRTUAL_HOST
16:16:52 	at org.apache.deltaspike.core.api.config.ConfigResolver$PropertyBuilder.withStringDefault(ConfigResolver.java:735)
16:16:52 	at org.apache.deltaspike.core.spi.config.BaseConfigPropertyProducer.getPropertyValue(BaseConfigPropertyProducer.java:127)
16:16:52 	at org.apache.deltaspike.core.spi.config.BaseConfigPropertyProducer.getStringPropertyValue(BaseConfigPropertyProducer.java:108)
16:16:52 	at org.apache.deltaspike.core.impl.config.DefaultConfigPropertyProducer.produceStringConfiguration(DefaultConfigPropertyProducer.java:43)
16:16:52 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
16:16:52 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
16:16:52 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
16:16:52 	at java.lang.reflect.Method.invoke(Method.java:606)
16:16:52 	at org.jboss.weld.injection.StaticMethodInjectionPoint.invoke(StaticMethodInjectionPoint.java:88)
16:16:52 	at org.jboss.weld.injection.StaticMethodInjectionPoint.invoke(StaticMethodInjectionPoint.java:78)
16:16:52 	at org.jboss.weld.injection.producer.ProducerMethodProducer.produce(ProducerMethodProducer.java:99)
16:16:52 	at org.jboss.weld.injection.producer.AbstractMemberProducer.produce(AbstractMemberProducer.java:161)
16:16:52 	at org.jboss.weld.bean.AbstractProducerBean.create(AbstractProducerBean.java:181)
16:16:52 	at org.jboss.weld.context.unbound.DependentContextImpl.get(DependentContextImpl.java:70)
16:16:52 	at org.jboss.weld.bean.ContextualInstanceStrategy$DefaultContextualInstanceStrategy.get(ContextualInstanceStrategy.java:101)
16:16:52 	at org.jboss.weld.bean.ContextualInstance.get(ContextualInstance.java:50)
16:16:52 	at org.jboss.weld.manager.BeanManagerImpl.getReference(BeanManagerImpl.java:742)
16:16:52 	at org.jboss.weld.manager.BeanManagerImpl.getInjectableReference(BeanManagerImpl.java:842)
16:16:52 	at org.jboss.weld.injection.FieldInjectionPoint.inject(FieldInjectionPoint.java:92)
16:16:52 	at org.jboss.weld.util.Beans.injectBoundFields(Beans.java:378)
16:16:52 	at org.jboss.weld.util.Beans.injectFieldsAndInitializers(Beans.java:389)
16:16:52 	at org.jboss.weld.injection.producer.ResourceInjector$1.proceed(ResourceInjector.java:70)
16:16:52 	at org.jboss.weld.injection.InjectionContextImpl.run(InjectionContextImpl.java:48)
16:16:52 	at org.jboss.weld.injection.producer.ResourceInjector.inject(ResourceInjector.java:72)
16:16:52 	at org.jboss.weld.injection.producer.BasicInjectionTarget.inject(BasicInjectionTarget.java:121)
16:16:52 	at org.jboss.weld.bean.ManagedBean.create(ManagedBean.java:159)
16:16:52 	at org.jboss.weld.context.unbound.DependentContextImpl.get(DependentContextImpl.java:70)
16:16:52 	at org.jboss.weld.bean.ContextualInstanceStrategy$DefaultContextualInstanceStrategy.get(ContextualInstanceStrategy.java:101)
16:16:52 	at org.jboss.weld.bean.ContextualInstance.get(ContextualInstance.java:50)
16:16:52 	at org.jboss.weld.manager.BeanManagerImpl.getReference(BeanManagerImpl.java:742)
16:16:52 	at org.jboss.weld.manager.BeanManagerImpl.getInjectableReference(BeanManagerImpl.java:842)
16:16:52 	at org.jboss.weld.injection.FieldInjectionPoint.inject(FieldInjectionPoint.java:92)
16:16:52 	at org.jboss.weld.util.Beans.injectBoundFields(Beans.java:378)
16:16:52 	at org.jboss.weld.util.Beans.injectFieldsAndInitializers(Beans.java:389)
16:16:52 	at org.jboss.weld.injection.producer.ResourceInjector$1.proceed(ResourceInjector.java:70)
16:16:52 	at org.jboss.weld.injection.InjectionContextImpl.run(InjectionContextImpl.java:48)
16:16:52 	at org.jboss.weld.injection.producer.ResourceInjector.inject(ResourceInjector.java:72)
16:16:52 	at org.jboss.weld.injection.producer.BasicInjectionTarget.inject(BasicInjectionTarget.java:121)
16:16:52 	at org.jboss.weld.bean.ManagedBean.create(ManagedBean.java:159)
16:16:52 	at org.jboss.weld.context.AbstractContext.get(AbstractContext.java:96)
16:16:52 	at org.jboss.weld.bean.ContextualInstanceStrategy$DefaultContextualInstanceStrategy.get(ContextualInstanceStrategy.java:101)
16:16:52 	at org.jboss.weld.bean.ContextualInstanceStrategy$ApplicationScopedContextualInstanceStrategy.get(ContextualInstanceStrategy.java:141)
16:16:52 	at org.jboss.weld.bean.ContextualInstance.get(ContextualInstance.java:50)
16:16:52 	at org.jboss.weld.bean.proxy.ContextBeanInstance.getInstance(ContextBeanInstance.java:99)
16:16:52 	at org.jboss.weld.bean.proxy.ProxyMethodHandler.getInstance(ProxyMethodHandler.java:125)
16:16:52 	at io.fabric8.mq.Fabric8MQ$Proxy$_$$_WeldClientProxy.getFabric8MQStatus(Unknown Source)
16:16:52 	at io.fabric8.mq.MainTest.doStart(MainTest.java:42)
```

**Comment by rawlingsj** [comment]
TL;DR thrown from deltaspike  `RuntimeException: Empty String or null supplied as string-default value for property VIRTUAL_HOST`

@iocanel don't suppose this means anything to you?

**Comment by davsclaus** [comment]
There was a deltaspike upgrade from an old 1.0.3 to 1.5.1 release. I guess they have improved delta spike to fail on that error, which from the message seems reasonable, an empty default value.

**Comment by davsclaus** [comment]
I think its this that just maybe need to remove the default value
https://github.com/fabric8io/fabric8-ipaas/blob/master/fabric8mq/src/main/java/io/fabric8/mq/Fabric8MQStatus.java#L47

**Comment by davsclaus** [comment]
There is a PR that fixes this
https://github.com/fabric8io/fabric8-ipaas/pull/84

**Comment by davsclaus** [comment]
retest this please

**Comment by davsclaus** [comment]
yay this has been merged now. I think @rawlingsj you can close this PR

**Comment by rawlingsj** [comment]
perfect thanks @davsclaus
